multicluster/test/e2e/service_test.go

```diff
@@ -47,32 +47,39 @@ func testServiceExport(t *testing.T, data *TestData) {
 // we create a nginx in on cluster(east), and try to curl it in another cluster(west).
 // If we got status code 200, it means that the resources is exported by the east cluster
 // and imported by the west cluster.
-// TODO(yang): reorg test function contents
-func (data *TestData) testServiceExport(t *testing.T) {
-	podName := randName("test-nginx-")
-	clientPodName := "test-service-client"
-
-	if err := createPodWrapper(t, data, westCluster, multiClusterTestNamespace, podName, nginxImage, "nginx", nil, nil, nil, nil, false, nil); err != nil {
-		t.Fatalf("Error when creating nginx Pod in west cluster: %v", err)
-	}
-	defer deletePodWrapper(t, data, westCluster, multiClusterTestNamespace, podName)
-
-	if err := createPodWrapper(t, data, eastCluster, multiClusterTestNamespace, podName, nginxImage, "nginx", nil, nil, nil, nil, false, nil); err != nil {
-		t.Fatalf("Error when creating nginx Pod in east cluster: %v", err)
-	}
-	defer deletePodWrapper(t, data, eastCluster, multiClusterTestNamespace, podName)
 
+func (data *TestData) setupTestServices(t *testing.T) {
```

Comment:
I think setting up test Pods should also be a part of this function, instead of in fixtures.go

Reply:
Got it, thanks! Will fix!